include/spirv-tools/libspirv.h

```diff
@@ -654,6 +654,11 @@ SPIRV_TOOLS_EXPORT void spvOptimizerOptionsSetPreserveBindings(
 SPIRV_TOOLS_EXPORT void spvOptimizerOptionsSetPreserveSpecConstants(
     spv_optimizer_options options, bool val);
 
+// Records whether all functions with DontInline flag within the module
+// should be preserved from inline pass.
+SPIRV_TOOLS_EXPORT void spvOptimizerOptionsSetPreserveDontInlineFunctions(
+    spv_optimizer_options options, bool val);
```

Comment:
Why do we need this at all?  If a function has the "no-inline" bit set, it should just not be inlined.  I'm not sure what advantage is there in weakening the "no-inline" bits this way.

Reply:
My original thought was that we may need an option to keep the behavior before this change.
I agree that a function with "DontInline" should just not be inline, I've already removed the option.